.github/actions/setup-and-build/action.yml

```
@@ -3,13 +3,31 @@ description: Checkout sources, install dependencies, build and prepare for tests
 
 inputs:
   enable_redis:
-    description: 'Enable Redis'
+    description: "Enable Redis"
     required: false
-    default: 'false'
+    default: "false"
 
 runs:
   using: "composite"
   steps:
+    - name: Checkout sources
+      uses: actions/checkout@v3
+
+    - name: Cache Rust dependencies
+      uses: buildjet/cache@v3
+      with:
+        path: |
+          ~/.cargo/registry
+          ~/.cargo/git
+          target
```

Can you check if putting

```suggestion
          **/target
```

works here? Or does this line already do the job without wildcarding.